learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -6,23 +6,24 @@ Visual Scripting
 Introduction
 ------------
 
-Visual Scripting is always a controversial topic, surrounded by many misunderstandings.
-Programmers often think the aim of it is replacing their beloved code, so they
-act defensively against it. Artists and game designers think that, with it,
-they can write any game they want with little effort without the need of programmers.
+Visual Scripting is a controversial topic, with many misunderstandings.
+Programmers often think the aim of Visual Scripting is to replace their beloved code.
+Artists and Game Designers tend to think that they can write any game they want with
+little effort without the need of programmers.
 
 This situation is the result of really bad marketing from many products over time 
-which, in their aim to capture the interest of artists and game designers, also
+which, in their aim to capture the interest of Artists and Game Designers, also
 made programmers feel like they are being replaced by visual tools.
 
-Truth is mostly, somewhat, in the middle. First of all, programmers can rest
-assured that Visual Scripting in no way makes code obsolete. Artists and game
-designers can also rest assured that they will be able to do a lot of things
-without programming.
+Truth is mostly, somewhat, in the middle.
+
+First of all, programmers can rest assured that Visual Scripting in no way makes
+code obsolete. Artists and Game Designers can also rest assured that they will be
+able to do a lot of things without programming.
```

Comment:
Shouldn't repeat  "rest assured" in two consecutive sentences.  Remove the "First of all" because there are only two groups being described here, and the second one isn't being referred to numerically.

Why do "Artists" and "Game Designers" get to be capitalized, but "programmers" don't?  None of these are proper nouns.